### PR TITLE
specs: require of e2fsprogs

### PR DESCRIPTION
**Pull request by guits**
in ceph/ceph-container we've realized that `e2fsprogs` isn't installed in
centos container image because ceph hasn't a dependency.
It has for consequence to fail when deploying a containerized cluster
with dmcrypt when using centos image.

Typical error encountered:

typical error:
```
......
get_dm_uuid: get_dm_uuid /dev/sda uuid path is /sys/dev/block/8:0/dm/uuid
get_dm_uuid: get_dm_uuid /dev/sda uuid path is /sys/dev/block/8:0/dm/uuid
get_dm_uuid: get_dm_uuid /dev/sda5 uuid path is /sys/dev/block/8:5/dm/uuid
populate: Creating lockbox fs on %s: mkfs -t ext4 /dev/sda5
command_check_call: Running command: /usr/sbin/mkfs -t ext4 /dev/sda5
mkfs.ext4: No such file or directory
Traceback (most recent call last):
  File "/usr/sbin/ceph-disk", line 9, in <module>
    load_entry_point('ceph-disk==1.0.0', 'console_scripts', 'ceph-disk')()
......
```

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by ktdreyer** [comment]
Does this need to be backported to Luminous? If so would you please file a ticket at http://tracker.ceph.com to track that?

**Comment by guits** [comment]
@ktdreyer I've filed this : https://tracker.ceph.com/issues/23650

**Comment by ktdreyer** [comment]
Thank you @guits !

File: ceph.spec.in [hunk 1]
```diff
@@ -295,6 +295,7 @@ Requires:      python%{_python_buildid}-requests
 Requires:      python%{_python_buildid}-setuptools
 Requires:      util-linux
 Requires:      xfsprogs
+Requires:      e2fsprogs
```
Comment (smithfarm) [inline ceph.spec.in]: The `Requires:` lines are in some semblance of alphabetical order. Could you move e2fsprogs up between "cryptsetup" and "findutils" please?

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-04-12_15:56:07-rados-wip-kefu-testing-2018-04-12-2211-distro-basic-mira/

failed tests are 

- tracked by http://tracker.ceph.com/issues/23699
- caused by #19117
- fixed by #21406

**Comment by alfredodeza** [comment]
This PR never got backported to luminous, and now distros that don't have `e2fsprogs` pre-installed are causing failures

**Comment by smithfarm** [comment]
> This PR never got backported to luminous, and now distros that don't have e2fsprogs pre-installed are causing failures

They weren't failing before, and now they're failing? I.e. it's a regression?

**Pull request merged**